static/js/components/ranking_unit.tsx

```diff
@@ -93,24 +109,50 @@ export function RankingUnit(props: RankingUnitPropType): JSX.Element {
                     text={point.placeName || point.placeDcid}
                   />
                 </td>
-                {!props.hideValue && (
+                {!props.hideValue && _.isEmpty(point.values) && (
                   <td className="stat">
                     <span
                       className={`num-value ${
                         point.placeDcid === props.highlightedDcid ? "bold" : ""
                       }`}
                     >
                       {formatNumber(
-                        props.scaling
-                          ? point.value * props.scaling
+                        props.scaling &&
+                          props.scaling.length &&
```

Comment:
nit: instead of checking props.scaling && props.scaling.length, can just check !_.isEmpty(props.scaling)? Same with props.unit

Reply:
done